starter-project-web-vue2/pages/liben/articles/_id/edit.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <v-container>
+    <v-row justify="center">
+      <v-col md="8" cols="12">
+        <h1>Edit Article</h1>
+        <LibenArticleForm :article="article" @on-submit="editArticle" />
```

Comment:
nit;(optional) since `LibenArticleForm` is not used elsewhere the content can be inserted instead of different component.